dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -8894,6 +8941,9 @@ CONFIG_ENCRYPTED_KEYS=y
 CONFIG_KEY_DH_OPERATIONS=y
 CONFIG_KEY_NOTIFICATIONS=y
 # CONFIG_SECURITY_DMESG_RESTRICT is not set
+CONFIG_PROC_MEM_ALWAYS_FORCE=y
```

Comment:
What is this doing?

Reply:
https://lore.kernel.org/all/20240726090858.71541-1-adrian.ratiu@collabora.com/

```
This adds a Kconfig option and boot param to allow removing
the FOLL_FORCE flag from /proc/pid/mem write calls because
it can be abused.

The traditional forcing behavior is kept as default because
it can break GDB and some other use cases.

Previously we tried a more sophisticated approach allowing
distributions to fine-tune /proc/pid/mem behavior, however
that got NAK-ed by Linus [1], who prefers this simpler
approach with semantics also easier to understand for users.

Link: https://lore.kernel.org/lkml/CAHk-=wiGWLChxYmUA5HrT5aopZrB7_2VTa0NLZcxORgkUe5tEQ@mail.gmail.com/ [1]
```

I think this restriction may even be good for us (?)

Reply:
If this enabled this option:
```
- 'always' traditional behavior always allows mem overrides.
```
Then it looks good for us. I would assume it still shouldn't corrupt kernel.

Reply:
Yes, CONFIG_PROC_MEM_ALWAYS_FORCE enables `always`.